lib/fastlane/plugin/wpmreleasetoolkit/helper/ci_helper.rb

```diff
@@ -0,0 +1,58 @@
+require 'fastlane_core/ui/ui'
+require 'net/http'
+require 'uri'
+require 'json'
+
+module CIHelper
+    def trig_job(branch, parameters=nil)
+      raise "Not implemented"
+    end
+
+    def login
+        raise "Not implemented"
+    end
+
+    def organization
+        raise "Not implemented"
+    end
+
+    def repository 
+        raise "Not implemented"
+    end
+end
+
+module Fastlane
+    module Helper
+        class CircleCIHelper 
+            include CIHelper
+
+            attr_accessor :login, :organization, :repository
+
+            def initialize(login, repository, organization = "wordpress-mobile")
+                @login = login
+                @organization = organization
+                @repository = repository
+            end 
+
+            def command_uri
+                URI.parse("https://circleci.com/api/v2/project/github/#{@organization}/#{@repository}/pipeline")
+            end
+
+            def trig_job(branch, parameters=nil)
```

Comment:
I suggest to use more explicit names, and avoid obscure or unclear method names.
```suggestion
            def trigger_job(branch, parameters=nil)
```

Also, it could be nice to have named ruby parameters, that makes the code more readable. You can do that by using `:` after the parameter name when declaring the method:
```suggestion
            def trigger_job(branch:, parameters: nil)
```
then update the call sites to `trigger_job(branch: 'develop', parameters: { … }`